### ISSUE TITLE
 not available

### ISSUE THREAD

**Issue by vscodeerrors** [Microsoft/vscode#54757]
Issue Id: <b>ffc5c5e6-9404-acca-2e75-4068d1fda0bc</b><br /><br />*Versions* <br />- 1.25.1<br />*Stack* <br />Error: not available<br />[/vs/workbench/services/preferences/common/preferencesModels.ts#518:97 (update)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/workbench/services/preferences/common/preferencesModels.ts#L518:97 (update))<br />[/vs/workbench/parts/extensions/electron-browser/extensionsActions.ts#691:74](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/workbench/parts/extensions/electron-browser/extensionsActions.ts#L691:74)<br />[/vs/base/browser/ui/list/listWidget.ts#890:30 (domNode)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/list/listWidget.ts#L890:30 (domNode))<br />[/vs/base/browser/ui/list/listWidget.ts#292:12 (_styleElement)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/list/listWidget.ts#L292:12 (_styleElement))<br />[/vs/base/browser/ui/breadcrumbs/breadcrumbsWidget.ts#100:7 (scrollLeft)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/breadcrumbs/breadcrumbsWidget.ts#L100:7 (scrollLeft))<br />[/vs/base/browser/ui/scrollbar/scrollableElement.ts#506:18 (_mouseIsOver)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/scrollbar/scrollableElement.ts#L506:18 (_mouseIsOver))<br />[/vs/base/browser/ui/scrollbar/scrollableElement.ts#443:7 (this)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/scrollbar/scrollableElement.ts#L443:7 (this))<br />[/vs/base/browser/ui/list/listView.ts#168:42 (onPointer)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/list/listView.ts#L168:42 (onPointer))<br />[/vs/base/browser/ui/list/listWidget.ts#509:9 (event)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/list/listWidget.ts#L509:9 (event))<br />[/vs/base/common/event.ts#277:16 (open)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/common/event.ts#L277:16 (open))<br />[/vs/base/browser/ui/list/listWidget.ts#504:14 (Object)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/list/listWidget.ts#L504:14 (Object))<br />[/vs/base/browser/ui/list/listWidget.ts#866:1](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/list/listWidget.ts#L866:1)<br />[/vs/workbench/electron-browser/shell.ts#139:1 (query)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/workbench/electron-browser/shell.ts#L139:1 (query))<br />[/vs/platform/extensionManagement/node/extensionGalleryService.ts#397:12](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/platform/extensionManagement/node/extensionGalleryService.ts#L397:12)<br />[/vs/base/common/winjs.base.js#10:0](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/common/winjs.base.js#L10:0)<br />    at Object.s.__generator.a.label [as next] (ou<br />

**Comment by vscodeerrors** [comment]
Here is another similar callstack 

Error: not available<br />[/vs/workbench/parts/extensions/node/extensionsWorkbenchService.ts#215:49 (getManifest)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/workbench/parts/extensions/node/extensionsWorkbenchService.ts#L215:49 (getManifest))<br />[/vs/workbench/parts/extensions/electron-browser/extensionsList.ts#191:12 (renderElement)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/workbench/parts/extensions/electron-browser/extensionsList.ts#L191:12 (renderElement))<br />[/vs/base/browser/ui/list/listPaging.ts#43:24 (renderElement)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/list/listPaging.ts#L43:24 (renderElement))<br />[/vs/base/browser/ui/list/listWidget.ts#816:12 (renderElement)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/list/listWidget.ts#L816:12 (renderElement))<br />[/vs/base/browser/ui/list/listView.ts#301:11 (insertItemInDOM)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/list/listView.ts#L301:11 (insertItemInDOM))<br />[/vs/base/browser/ui/list/listView.ts#195:9 (_splice)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/list/listView.ts#L195:9 (_splice))<br />[/vs/base/browser/ui/list/listView.ts#137:15 (splice)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/list/listView.ts#L137:15 (splice))<br />[/vs/base/browser/ui/list/listWidget.ts#36:14 (splice)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/list/listWidget.ts#L36:14 (splice))<br />[/vs/base/browser/ui/list/listWidget.ts#946:56 (fn)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/list/listWidget.ts#L946:56 (fn))<br />[/vs/base/common/event.ts#403:2 (bufferEvents)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/common/event.ts#L403:2 (bufferEvents))<br />[/vs/base/browser/ui/list/listWidget.ts#946:21 (splice)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/list/listWidget.ts#L946:21 (splice))<br />[/vs/base/browser/ui/list/listPaging.ts#126:12 (model)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/browser/ui/list/listPaging.ts#L126:12 (model))<br />[/vs/workbench/parts/extensions/electron-browser/extensionsViews.ts#550:21 (setModel)](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/workbench/parts/extensions/electron-browser/extensionsViews.ts#L550:21 (setModel))<br />[/vs/workbench/parts/extensions/electron-browser/extensionsViews.ts#112:7](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/workbench/parts/extensions/electron-browser/extensionsViews.ts#L112:7)<br />[/vs/base/common/winjs.base.js#10:0](https://github.com/microsoft/vscode/blob/1dfc5e557209371715f655691b1235b6b26a06be/src/vs/base/common/winjs.base.js#L10:0)<br />    at Object.s.__generator.a.label [as next] (ou<br />

**Comment by mjbvz** [comment]
Hit this wile testing. Reliable repo for me: 

1. Open extensions view and search `Forcecode`

**Comment by ramya-rao-a** [comment]
Looks like the ForceCode extension was not packaged using our vsce tool and thus the missing manifest.
We introduced the checking of manifest in `extensionsList.ts` in the last milestone, and so when searching for ForceCode, the error is fired.

Fixed it by not bubbling up the error when trying to fetch the manifest from the extensions view.

This error can still occur (with a different call stack of course) when the user clicks on such extension. This is because the extension editor tries to fetch the manifest as well.

Assigning to @sandy081 for further calls on how we want to deal with extensions that do not have a manifest.
